aws-rotate-keys/executor/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM circleci/python:3-stretch
+
```

Comment:
Does this run as root?

Reply:
it runs as `circleci` just like in he parent image